README.md

```diff
@@ -1,23 +1,41 @@
+<!-- LTex: enabled=false -->
+
 # just-the-docs-template
 
-This is a *bare-minimum* template to create a [Jekyll] site that:
+## Skip the details, get me going!
 
-- uses the [Just the Docs] theme;
-- can be built and published on [GitHub Pages];
-- can be built and previewed locally, and published on other platforms.
+<div align="center">
 
-More specifically, the created site:
+[![Open in Dev Containers](https://img.shields.io/static/v1?label=Dev%20Container&message=Open&color=blue&logo=visualstudiocode&style=for-the-badge)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/just-the-docs/just-the-docs-template)
+[![Open in Codespaces](https://img.shields.io/static/v1?label=CodeSpaces&message=Launch&color=blue&logo=github&style=for-the-badge)](https://github.com/codespaces/new/just-the-docs/just-the-docs-template)
 
-- uses a gem-based approach, i.e. uses a `Gemfile` and loads the `just-the-docs` gem;
-- uses the [GitHub Pages / Actions workflow] to build and publish the site on GitHub Pages.
+</div>
 
-To get started with creating a site, just click "[use this template]"!
+The **DevContainer** requires VSCode and Docker to be installed on your machine.
 
-If you want to maintain your docs in the `docs` directory of an existing project repo, see [Hosting your docs from an existing project repo](#hosting-your-docs-from-an-existing-project-repo).
+The **CodeSpace** requires nothing of that and is the ideal solution for beginners.
```

Comment:
Can we add links to these services; for the latter, I think we should explain that this is a GitHub product.